README.rst

```diff
@@ -19,7 +19,43 @@ bugzilla and who get the notifications for changes in the git, builds or bugs.
 
 
 Hacking
--------
+=======
+
+Hacking with Vagrant
+--------------------
+Quickly start hacking on pkgdb2 using the vagrant setup that is included in the
+pkgdb2 repo is super simple.
+
+First, install Vagrant and the vagrant-libvirt plugin from the official Fedora
+repos::
+
+    $ sudo yum install vagrant vagrant-libvirt
+
+The pkgdb2 vagrant setup uses vagrant-sshfs for syncing files between your host
+and the vagrant dev machine. vagrant-sshfs is not in the Fedora repos (yet), so
+we install the vagrant-sshfs plugin from dustymabe's COPR repo::
+
+    $ sudo dnf copr enable dustymabe/vagrant-sshfs
+    $ sudo dnf install vagrant-sshfs
+
+Now, from within main directory (the one with the Vagrantfile in it) of your git
+checkout of pkgdb, run the ``vagrant up`` command to provision your dev
+environment::
+
+    $ vagrant up
+
+When this command is completed (it may take a while) you will be able to ssh
+into your dev VM with ``vagrant ssh`` and then run the command to start the
+pkgdb2 server::
+
+    $ vagrant ssh
+    [vagrant@localhost ~]$ pushd /vagrant/; ./runserver.py -c pkgdb2/vagrant_default_config.py --host \"0.0.0.0\";
```

Comment:
Whoops! I copy pasted from the same text in the vagrantfile, and forgot to remove the escaping. 

Fixed now.